src/nft/types/discover/react-table-config.d.ts

```diff
@@ -2,19 +2,23 @@ import { UseSortByColumnOptions, UseSortByColumnProps, UseSortByOptions, UseSort
 
 /* https://github.com/TanStack/table/issues/2970 */
 declare module 'react-table' {
+  // eslint-disable-next-line import/no-unused-modules
```

Comment:
You should instead be disabling this rule for .d.ts files. Is that possible?

Reply:
yes